charts/oidc-webhook-authenticator/charts/runtime/values.yaml

```diff
@@ -58,3 +58,7 @@ serviceAccountTokenVolumeProjection:
 
 # List with identifiers of the API. Tokens used against the API should be bound to at least one of these audiences.
 apiAudiences: []
+
+additionalLabels:
+  hpa: []
+  deployment: []
```

Comment:
Is there a use case that requires deployment and hpa to have different additional labels?
If not, I would propose to merge them into single configuration `additionalLabels`.

Also, why are the labels list of something? They can be just a map `{}`.

Reply:
> They can be just a map {}.

You are right - your suggestion sounds better.


> Is there a use case that requires deployment and hpa to have different additional labels?

Not right now, but still I prefer that they are handled separately so we do not change code if this requirement occurs.